File: 1352C. K-th Not Divisible by n.js
function getKthNonDivisibleByN(n,k) {
  var p = n-1
	var mod = k % p
	var d = Math.floor(k/p)
	var ans = d*n+mod
	if (k % p == 0)
		ans = ans-1
	return ans
}

var tc = Number(readline())
while (tc--) {
	var str = readline()
	var arr = str.split(" ")
	var n = arr[0]
	var k = arr[1] 
  print(getKthNonDivisibleByN(n,k))
}

File: 1352C. K-th Not Divisible by n.py
def getKthNonDivisibleByN(n,k):
	p = n-1
	mod = k % p
	d = k // p
	ans = d*n+mod
	if k % p == 0:
		ans = ans-1
	return ans

tc = int(input())
for pos in range(tc):
	n,k = input().split(" ")
	n,k = [int(n),int(k)]
	print(getKthNonDivisibleByN(n,k))